cve_bin_tool/util.py

```diff
@@ -79,13 +79,21 @@ def regex_find(lines, version_patterns) -> str:
     """Search a set of lines to find a match for the given regex"""
     new_guess = ""
 
-    for line in lines:
-        for pattern in version_patterns:
-            match = pattern.search(line)
+    for pattern in version_patterns:
+        "Multiline Regex Support"
```

Comment:
Why is this just a string? Shouldn't it be a comment?

Reply:
got mixed with docstring and comment 😅